drivers/hmis/app/models/hmis/client_alert.rb

```diff
@@ -0,0 +1,10 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::ClientAlert < Hmis::HmisBase
```

Comment:
looks like this model has a deleted_at attribute so it seems like we want to use soft delete. If so, I think we'd need to include the behavior from the `paranoia` gem
```suggestion
class Hmis::ClientAlert < Hmis::HmisBase
  acts_as_paranoid
```